modules/node_modules/@ncigdc/utils/download/index.js

```diff
@@ -5,15 +5,29 @@ import _ from 'lodash';
 import $ from 'jquery';
 import Cookies from 'js-cookie';
 
+import { store } from 'webpack_entry/Portal';
+import { setModal } from '@ncigdc/dux/modal';
+import { Row, Column } from '@ncigdc/uikit/Flex';
+import Button from '@ncigdc/uikit/Button';
+
 // const cookiePath = document.querySelector('base').getAttribute('href')
 const cookiePath = '/';
 const iFrameIdPrefix = '__downloader_iframe__';
 const formIdPrefix = '__downloader_form__';
 const getIframeResponse = iFrame => JSON.parse(iFrame.contents().find('body pre').text());
 const showErrorModal = error => {
   const warning = error.warning || error.message;
-
-  // open modal
+  store.dispatch(setModal(
```

Comment:
I think this util is used across the site, so bam slicing specific stuff probably shouldn't be in it?